lib/github/hooks/issue.ts

```diff
@@ -427,6 +426,99 @@ export const onPullRequestOpened = async (webhooks: Webhooks) => {
   });
 };
 
+export const onPullRequestMerged = async (webhooks: Webhooks) => {
+  webhooks.on(EVENT_TRIGGERS.PULL_REQUEST_CLOSED, async (context) => {
+    const pullRequest = context.payload.pull_request;
+    const repo = context.payload.repository.name;
+    const owner = context.payload.repository.owner.login;
+
+    // Early exit if the pull request was not merged
+    if (!pullRequest.merged) {
+      console.log("Pull request was not merged.");
+      return;
+    }
+
+    const octokit = getOctokitInstance(context.payload.installation?.id!);
+
+    // Parse the pull request body to find linked issues (e.g., " #123")
+    const issueNumbers = extractIssueNumbersFromPrBody(pullRequest.body!);
+    if (!issueNumbers.length) {
+      console.log("No issues are mentioned in the pull request.");
+      return;
+    }
+
+    // Check repository enrollment first
+    const ossGgRepo = await getRepositoryByGithubId(context.payload.repository.id);
+    if (!ossGgRepo) {
+      console.log("Repository is not enrolled in oss.gg.");
+      return;
+    }
+
+    // Loop through all the extracted issue numbers
+    for (const issueNumber of issueNumbers) {
+      try {
+        // Fetch the issue details using the GitHub API
+        const { data: issue } = await octokit.issues.get({
+          owner,
+          repo,
+          issue_number: issueNumber,
+        });
+
+        // Get the labels of the linked issue
+        const issueLabels = issue.labels.map((label: { name: string }) => label.name);
+
+        // Check if the "🕹️ oss.gg" label is present, skip if not
+        const ossLabel = issueLabels.find((label) => label.includes(OSS_GG_LABEL));
+        if (!ossLabel) {
+          console.log(`Issue #${issueNumber} does not have the 🕹️ oss.gg label. Skipping.`);
+          continue;  // Skip this issue and move to the next one
+        }
+
+        // Extract points from labels like "🕹️ 50 points", "🕹️ 100 points", etc.
```

Comment:
we have a `extractPointsFromLabel` helper function in github/utils - please use this